thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/views/AnalysisView.js

```diff
@@ -39,18 +41,27 @@ AnalysisView.prototype = {
       }
     });
 
-    analysisMetricSelect.on("change", function(e) {
-      var selectedElement = $(e.currentTarget);
-      var selectedData = selectedElement.select2("data");
-      var metricId = selectedData.map(function (e) {return e.id})[0];
-      var metricAlias = selectedData.map(function (e) {return e.text})[0];
-      var metricName = selectedData.map(function (e) {return e.name})[0];
-      self.viewParams['metric'] = {id: metricId, alias: metricAlias, allowClear:true, name:metricName};
+    analysisMetricSelect.on('change', (e) => {
+      const selectedElement = $(e.currentTarget);
```

Comment:
We generally by convention prefix $ elements with $, so `$selectedElement`, so it's clear to the reader later in the context that this is $ element as opposed to a DOM element